exporter/awsxrayexporter/go.mod

```diff
@@ -4,12 +4,10 @@ go 1.14
 
 require (
 	github.com/aws/aws-sdk-go v1.35.2
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/awsxray v0.0.0-00010101000000-000000000000
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/awsxray v0.11.1-0.20200928205243-e3493cedf4b6
```

Comment:
With this change it will be impossible to change something in the internal/awsxray and exporter/receiver. Also it will always be outdated.

Reply:
Understand it now. Do you think it can be helpful if we add the comments in xray receiver/exporter for anyone may also reference xray components alone in their package? Otherwise, I can close this PR.